pymc/tests/test_distributions_moments.py

```diff
@@ -897,4 +898,12 @@ def test_asymmetriclaplace_moment(b, kappa, mu, size, expected):
 def test_rice_moment(nu, sigma, size, expected):
     with Model() as model:
         Rice("x", nu=nu, sigma=sigma, size=size)
+
+
+@pytest.mark.parametrize(
+    "alpha, K, size, expected", [(3, 11, None, (3 / 4) ** np.arange(12) * 1 / 3)]
+)
+def test_stickbreakingweights_moment(alpha, K, size, expected):
```

Comment:
Here, I am obtaining the following error and I am not sure where `result` comes from. Does it represent a random draw? My understanding is that, for other moment tests, `result` is not a random draw. I obtain the same error with the same numbers every time I try to run this test.

<img width="1273" alt="Screen Shot 2021-11-17 at 5 44 19 PM" src="https://user-images.githubusercontent.com/24764859/142294439-f30e88ce-433c-4ab8-b816-403281d9e962.png">

Reply:
`result` is the `moment`. It seems to deviate slightly from the expected moment. How did you pre-compute it? Are you sure it matches the `get_moment` computation?